kubetest2-gke/deployer/build_test.go

```diff
@@ -70,7 +112,7 @@ func TestNormalizeVersion(t *testing.T) {
 		{
 			name:            "alpha version with patch",
 			version:         "1.16.13-alpha.123",
-			expectedVersion: "1.16.13-gke.123+alpha",
+			expectedVersion: "1.16.13-gke.99.0+alpha.123",
```

Comment:
This behavior is changing, are we sure we don't want `.123.0` instead?

Reply:
yeah, alpha.123 has no real bearing on gke build version... preserving the existing suffix intact makes more sense to me

Reply:
sgtm

/hold cancel